build.gradle

```diff
@@ -57,7 +57,7 @@ ext {
 dependencies {
     implementation 'com.google.code.gson:gson:2.8.9'
     implementation 'org.json:json:20210307'
-    implementation 'org.mifos:ph-ee-connector-common:1.4.1-SNAPSHOT'
+    implementation 'org.mifos:ph-ee-connector-common:0.0.0'
```

Comment:
Why 0.0.0?

Reply:
That is only for CI to pass. Once The common-connector PR is merged and the artifact is published I will update the version accordingly.

Reply:
ok

Reply:
This PR is now using `1.10.0` as connector common [PR](https://github.com/fynarfin/ph-ee-connector-common/pull/1) got merged.